app/assets/stylesheets/homepage.css.scss

```diff
@@ -16,14 +16,39 @@
     font-weight: 200;
 }
 
-.nav-bar-nk {
+#logo {
+    text-decoration: none;
+    margin-left: -70px ;
+}
+
+.nav-bar-link {
     color: white;
-    font-size: 20px;
+    font-size: 1.1em;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered color, display, font-size, height, padding, text-decoration